### DISCUSSION TITLE
Не загружаются файлы в заканчиваем мама проект

### DISCUSSION THREAD

**Discussion by comrade-che** [open-tutorials/cypress#73] [Q&A] [answered]
Система выдает ошибку при попытке загрузить файлы командой BASE_URL=https://raw.githubusercontent.com/open-tutorials/cypress/main node download.js

![1](https://user-images.githubusercontent.com/118734363/221874478-47682199-8bf4-48eb-97e9-648e62e37835.png)

Предварительно при установке wget система выдает сообщение о 6 уязвимостях(((

![2](https://user-images.githubusercontent.com/118734363/221874525-cef76b36-9325-4719-8de3-c53f7365c707.png)

![3](https://user-images.githubusercontent.com/118734363/221874546-a9ba53bb-111b-4b41-abc2-db613089e2c2.png)

![4](https://user-images.githubusercontent.com/118734363/221874562-6a1032d3-7768-4906-af1c-9ab72ddcc3ce.png)

![5](https://user-images.githubusercontent.com/118734363/221874576-ec2a6d2a-9549-49f7-bc52-d1ae061c23d6.png)

**Comment by Vladimir-Tarasov** [comment]
В ручную проверять правильно ли указан адрес источника (откуда должен файл скачиваться) пробовали? Открывается правильный файл?

**Reply by comrade-che** [reply]
Нет. но попробовал. Там 404

**Reply by Vladimir-Tarasov** [reply]
Cкорее всего неправильно указан адрес источника откуда должна производиться загрузка

**Reply by comrade-che** [reply]
там как бы и base url 404 выдает. Но markdown скачивает...

**Reply by Vladimir-Tarasov** [reply]
А по адресу, который указан для скачивания, в этот маркдаун в ручную перейти можно?

**Reply by comrade-che** [reply]
Нет
Вот такой он по туториалу - https://raw.githubusercontent.com/open-tutorials/cypress/main/test_cases

Вот такой он по факту - https://github.com/open-tutorials/cypress/tree/main/test_cases

Но скачивается же каким-то хреном)))

**Comment by Vladimir-Tarasov** [comment]
```

echo $BASE_URL/адрес файла
```
Попробуйте это использовать для проверки правильности формирования полного URL файла

**Reply by comrade-che** [reply]
Ок спасибо. попробую

**Reply by comrade-che** [reply]
Если честно до конца не понимаю ответа. Но так как ошибок нет. Значит путь правильный))))

![image](https://user-images.githubusercontent.com/118734363/222366399-4c6b15cd-cb68-4076-a1b4-4f04f14eb137.png)

**Reply by Vladimir-Tarasov** [reply]
Ну если попробовать перейти по этой URL, то должен открыться данный файл на github

**Reply by breslavsky** [reply]
Команду вводите не верно. В туториале нет `echo` для этой команды в начале.

**Reply by comrade-che** [reply]
Не, я просто эхом проверил доступность, как @Vladimir-Tarasov советовал)))

**Comment by breslavsky** [comment]
Предлагаю на стендап, странная ошибка.

**Reply by comrade-che** [reply]
Окей. Принято. Перенесу лыжную тренировку на пораньше)))

**Comment by comrade-che** [comment, marked as answer]
Вопрос можно закрыт. Все работает. Мой косяк
